packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -150,12 +150,6 @@ export class VaTextInput {
    */
   @Prop({reflect: true}) uswds?: boolean = false;
 
-  /**
-   * Whether the component should show a character count message.
-   * Has no effect without uswds and maxlength being set.
-   */
-  @Prop() charcount?: boolean = false;
```

Comment:
Is this being used anywhere in vets-website at the moment?

Reply:
No, it's not being used in vets-website anywhere.